code/main/edge/edgeRequestWorker.brs

```diff
@@ -82,8 +86,11 @@ function _adb_EdgeRequestWorker() as object
                 if networkResponse.isSuccessful()
                     edgeResponse = _adb_EdgeResponse(requestId, networkResponse.getResponseCode(), networkResponse.getResponseString())
                     responseArray.Push(edgeResponse)
+                    ' Request sent out successfully
+                    m._lastFailedRequestTS = m._INVALID_WAIT_TIME
                 else if networkResponse.isRecoverable()
-                    _adb_logError("processRequests() - Edge request failed with recoverable error. Request will be retried.")
+                    m._lastFailedRequestTS = _adb_timestampInMillis()
+                    _adb_logError("processRequests() - Edge request failed with recoverable error:(" + networkResponse.getResponseString() + "). Request will be retried.")
```

Comment:
_adb_logWarning("processRequests() - Edge request failed with recoverable error code:(" + networkResponse.getResponseCode() + "). Request will be retried after " + StrI(_RETRY_WAIT_TIME_MILLIS) + " ms.")